site/helm-charts.md

```diff
@@ -16,6 +16,27 @@ The WebLogic Kubernetes Operator uses Helm to create and deploy any necessary re
 
 Helm has two parts: a client (Helm) and a server (Tiller). Tiller runs inside of your Kubernetes cluster, and manages releases (installations) of your charts.  See https://github.com/kubernetes/helm/blob/master/docs/install.md for detailed instructions on installing Helm and Tiller.
 
+In order to use Helm to install and manage the operator, you need to ensure that the Service Account that Tiller uses
```

Comment:
@rosemarymarano I believe that we've been consistently *not* capitalizing K8s resource types, correct?  So "Service Account" -> "service account"

Reply:
Correct.